app/src/main/java/org/oppia/android/app/recyclerview/RecyclerViewBindingAdapter.java

```diff
@@ -1,6 +1,8 @@
 package org.oppia.android.app.recyclerview;
 
 import android.graphics.drawable.Drawable;
+import android.util.Log;
```

Comment:
Please remove if this isn't needed. I'm surprised this didn't trigger a lint failure.

Reply:
Done